src/components/Axe.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const Axe = () => {
+  // Check we are not in production, axe should only be used during development.
+  if (typeof window !== 'undefined' && process.env.NODE_ENV !== 'production') {
```

Comment:
Would it be safer to check for development and/or test only?

Reply:
I think that could be useful, better to check that something is true, than isn't, I'll change `production` to `development` then.

Reply:
I've now updated this so if you agree with it, please `resolve this conversation.